agent/bench-scripts/gold/pbench-trafficgen/test-26.txt

```diff
@@ -0,0 +1,249 @@
+  decapsulate them and send them out the first device.
+
+--flow-mods=str[,str] (default is src-ip,dst-ip,src-mac,dst-mac)
+  A list of IP packet header fields which are used to implement unique flows.
+  All fields listed here are changed in unison, for example if "src-ip,src-mac"
+  are used, the first packet will have a source IP of 192.168.0.X and source MAC
+  of 01:02:03:04:X:06.  The second packet will have a source IP of
+  192.168.0.(X+1) and source MAC of 01:02:03:04:(X+1):06. The list of available
+  flow mods are:
+
+    src-ip,dst-ip,src-mac,dst-mac,src-port,dst-port,encap-src-mac,encap-dst-mac,
+    encap-src-ip,encap-dst-ip,protocol,none
+
+  Note that none is a special flow mod option.  It's presence will override and
+  disable all other specified flow mods
+  Note that the encap-* flow mods are only viable for overlay network tests like
+  VxLAN, and alter the inner packet header.
+  The maximum number of unique flows may be limited by the traffic-generator used.
+  For trex-txrx, it is 16.7 million.  Trex-txrx modifies the 2nd, 3rd, and 4th
+  octet in the IP addresses and the 3rd, 4th, and 5th octet in the MAC addresses
+  When using source or destination ports numbers, the range is limited to 1-32768
+  and higher flow counts will "roll" over back to 1
+
+--packet-protocol=str (default is UDP)
+  The protocol to use when generating IP packets.  Available options are TCP or
+  UDP.
+
+--trex-use-ht
+  Should TRex be allowed to use Hyperthreading siblings.  Defaults to disabled.
+--trex-use-l2
+  Should TRex be configured in L2 mode instead of L3.  Defaults to disabled.
+
+
+trafficgen debug options
+-----------------------------------------------
+
+--skip-trex-install
+  Do no install TRex server process (assumes you have one) installed already
+
+--skip-trex-server
+  Do no kill existing or start a new TRex server process (assumes you have one)
+  running already)
+
+--skip-git-pull
+  Do not call git pull on the lua-trafficgen repo if it already exists
+
+
+options common in most pbench benchmark scripts
+-----------------------------------------------
+
+--config=str
+  Name of the test config (i.e. jumbo_frames_and_network_throughput)
+
+--samples=int
+  The number of times each different test is run (to compute average)
+
+--max-failures=int
+  The maximm number of failures to get below stddev
+
+--max-stddev=int
+  The maximm percent stddev allowed to pass
+
+--postprocess-only=y|n
+  Don't run the benchmark, but postprocess data from previous test
+
+--run-dir=str
+  Optionally specify what directory should be used (usually only used if
+  postprocess-only=y)
+
+--start-iteration-num=int
+  Optionally skip the first (n-1) tests
+
+--tool-group=str
+  Start/stop/post-process tools using this group
+
+--pre-sample-cmd=str
+  run this command before executing each test sample
+
+--sysinfo=str,
+  comma seperated values of sysinfo to be collected, default="none"
+    available: default,none,all,a,b,c,d,e
```

Comment:
I presume this uses a mock version of pbench-collect-sysinfo?

Reply:
Yes, I believe so.